packages/common/src/store/buy-usdc/sagas.ts

```diff
@@ -173,6 +227,13 @@ function* doBuyUSDC({
       return
     }
 
+    // Transfer from the root wallet to the userbank
+    yield* call(transferStep, {
```

Comment:
For AUDIO purchases, we had the concept of recovering transactions. I left all of that out for USDC because we weren't doing a multi-step process. Now that we are, do we need to consider adding recovery? :-/

Reply:
We will want some recovery here I think, but intentionally left it out for now! Made this ticket: https://linear.app/audius/issue/PAY-1877/add-recovery-flow-for-purchasing-usdc